crates/lib/src/command/supervisor.rs

```diff
@@ -457,12 +453,13 @@ pub struct SupervisorId(NonZeroU64);
 
 impl Default for SupervisorId {
 	fn default() -> Self {
+		use std::{
+			collections::hash_map::RandomState,
+			hash::{BuildHasher, Hasher},
+		};
 		// generates pseudo-random u64 using [xorshift*](https://en.wikipedia.org/wiki/Xorshift#xorshift*)
+		let mut seed = RandomState::new().build_hasher().finish();
```

Comment:
Could we use RandomState to get the u64 directly without implementing xorshift ourselves on top?

Reply:
oh, honestly, you are so right, or at least i could not think of a good reason why not! can you? that was silly to keep haha!